amundsen_application/static/css/_variables-default.scss

```diff
@@ -62,6 +62,19 @@ $badge-primary-color: $cyan10;
 $badge-success-color: $mint20;
 $badge-warning-color: $amber30;
 
+$badge-negative-color: $sunset20;
+$badge-neutral-color: $gray20;
+$badge-positive-color: $mint20;
+$badge-warning-color: $amber30;
```

Comment:
The reason why I had not changed it is because this is the change that would break any custom configuration an implementation has for badges. I can go ahead with the change but I think we might need to let people know in case they are using it.

Reply:
You should be able to change them because the style users are supposed to set is a enumerable value, so `BadgeStyle.DANGER`. So although we might not be able to remove that enum itself, it's string value defined [here](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config-types.ts#L136-L143) can be updated to `'negative'` for all usages and that wouldn't break any existing configurations, but allow us to clean up our style variables.